client/router.js

```diff
@@ -34,6 +34,27 @@ FlowRouter.route('/legalnotice', {
 });
 
 
+FlowRouter.route( '/verify-email/:token', {
+    name: 'verify-email',
+    action( params ) {
+        Accounts.verifyEmail( params.token, ( error ) =>{
+            if ( error ) {
+                alert(error.reason);
```

Comment:
We could use the FlashMessage-class to display the error as well as the success message. Maybe we should also redirect to the home page even in the case of an error?